components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.persistence/src/org/wso2/integrationstudio/gmf/esb/internal/persistence/HTTPEndPointTransformer.java

```diff
@@ -100,21 +100,38 @@ public org.apache.synapse.endpoints.HTTPEndpoint create(HTTPEndpoint visualEndPo
 
         switch (visualEndPoint.getAuthType()) {
         case OAUTH:
+            HTTPEndpointOAuthAuthenticationMode oAuthAuthenticationMode = visualEndPoint.getOAuthAuthenticationMode();
+            String authMode = "";
+            if (oAuthAuthenticationMode.getValue() == HTTPEndpointOAuthAuthenticationMode.HEADER_OAUTH_AUTHENTICATION_MODE_VALUE) {
+                authMode = "header";
+            }else if (oAuthAuthenticationMode.getValue() == HTTPEndpointOAuthAuthenticationMode.HEADER_OAUTH_AUTHENTICATION_MODE_VALUE) {
```

Comment:
Formatting issue: " " is missing

Reply:
Aren't we evaluating the same expression here?

Reply:
Fixed in https://github.com/wso2/integration-studio/pull/1114